spec/helpers/chat_mirgation_message_spec.rb

```diff
@@ -0,0 +1,26 @@
+RSpec.describe ChatMigrationMessage, type: :helper do
+  describe '#display_message?' do
+
+    context 'before display_message date' do
+      it 'should be false' do
```

Comment:
I would change all the it description to 'returns true'  /false/nil. E.g: http://betterspecs.org/#should